### PR TITLE
chore: pnpm@8.0.0

### PR DESCRIPTION
**Pull request by juliusmarminge**
Closes #

## 🎯 Changes

What changes are made in this PR? Is it a feature or a bug fix?

## ✅ Checklist

- [ ] I have followed the steps listed in the [Contributing guide](https://github.com/trpc/trpc/blob/main/CONTRIBUTING.md).
- [ ] If necessary, I have added documentation related to the changes made.
- [ ] I have added or updated the tests related to the changes made.

### CODE REVIEW

File: .github/workflows/examples-old-nodes.yml [hunk 1]
```diff
@@ -73,3 +73,74 @@ jobs:
         run: pnpm turbo --filter ./examples/${{ matrix.dir }} test-dev
       - name: Run test-start
         run: pnpm turbo --filter ./examples/${{ matrix.dir }} test-start
+
+  e2e-old-nodes-node-14:
```
Comment (juliusmarminge) [inline .github/workflows/examples-old-nodes.yml:77]: is this worth it to retain the node-14 workflow?

pnpm@8 is >=node16 so i split this workflow into 2

Comment (sachinraja) [reply .github/workflows/examples-old-nodes.yml:77]: node 14 goes end of life next month so I think it's ok...

Comment (juliusmarminge) [reply .github/workflows/examples-old-nodes.yml]: aight i'll just nuke node14 then :)

File: scripts/addPrismaOverrides.ts [hunk 2]
```diff
@@ -37,12 +37,12 @@ async function main() {
...
           `${pkgJson.name}>@prisma/client`
         ] = `https://registry.npmjs.com/@prisma/client/-/client-${version}.tgz?id=${encodeURIComponent(
-          pkgJson.name,
+          pkgJson.name.replace(/\/|@/g, ''), // remove @ and / before. vitest hiccups otherwise
         )}`;
         rootPkgJson.pnpm.overrides[
           `${pkgJson.name}>prisma`
         ] = `https://registry.npmjs.com/prisma/-/prisma-${version}.tgz?id=${encodeURIComponent(
-          pkgJson.name,
+          pkgJson.name.replace(/\//g, ''),
```
Comment (kamilogorek) [inline scripts/addPrismaOverrides.ts:45]: Missed `@`?

Comment (juliusmarminge) [reply scripts/addPrismaOverrides.ts:45]: There is no @ in the `prisma` package, only `@prisma/client`

Comment (kamilogorek) [reply scripts/addPrismaOverrides.ts:45]: But `pkgJson.name` here is the name of the `example/*/package.json` files, not `prisma` nor `@prisma/client` 🤔

Comment (juliusmarminge) [reply scripts/addPrismaOverrides.ts:45]: Oh right - duh... not sure why its working then 🤷‍♂️🤷‍♂️🤷‍♂️ vitest was only complaining about the former i think 🙄🙄

**Pull request merged**